sdks/urbackend-python/test_sdk.py

```
@@ -0,0 +1,380 @@
+
+def separator(title: str) -> None:
+    print(f"\n{'=' * 60}")
+    print(f"  {title}")
+    print(f"{'=' * 60}")
+
+
+def main():
+    # ── 1. CONNECT ──────────────────────────────────────────────
+    separator("1. Connect")
+    client = UrBackendClient(api_key=API_KEY)
+    print(f"Client created with default base URL")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)